frontend/src/types/plan.ts

```diff
@@ -17,3 +17,17 @@ export enum PlanType {
 export type PlanPatch = {
   type: PlanType;
 };
+
+export interface Plan {
+  // Plan meta data
+  type: PlanType;
+  trialDays: number;
+  unitPrice: number;
+  trialPrice: number;
+  freeInstanceCount: number;
+  pricePerInstancePerMonth: number;
+  // Plan desc and feature
+  title: string;
+  description: string;
+  features: { id: string; content?: string; type?: string }[];
```

Comment:
Not sure the use of type

If it does get used, maybe we can refine it as enum like  "Counter" | "xxxx"

Reply:
Good point, it's used for old UX design, I'll remove it